pkg/standalone/common.go

```diff
@@ -44,9 +46,64 @@ func binaryFilePath(binaryDir string, binaryFilePrefix string) string {
 }
 
 func DefaultComponentsDirPath() string {
-	return path_filepath.Join(defaultDaprDirPath(), defaultComponentsDirName)
+	return path_filepath.Join(defaultDaprDirPath(), utils.DefaultComponentsDirName)
+}
+
+func DefaultResourcesDirPath() string {
+	return path_filepath.Join(defaultDaprDirPath(), utils.DefaultResourcesDirName)
+}
+
+// DefaultResourcesDirPrecedence returns the path to the resources directory if it exists, or otherwise the components directory.
+// TODO: Remove this function and use `DefaultResourcesDirPath` when `--components-path` flag is removed.
+func DefaultResourcesDirPrecedence() string {
```

Comment:
```suggestion
// GetResourcesDir returns the path where the resources are stored.
func GetResourcesDir() string {
    // TODO: clean up this logic and return `DefaultResourcesDirPath` when `--components-path` flag is removed.
```

This will abstract the logic for getting the resources dir. We can try to unexport `DefaultResourcesDirPath` and `DefaultComponentsDirPath`.